packages/statsmodels/meta.yaml

```diff
@@ -9,6 +9,9 @@ source:
     - patches/fix-scipy-blas-cythonize.patch
 
 build:
+  script: |
+    # Get rid of files with commas in their name, it breaks the wheel package.
+    find -name '*,*' -delete
```

Comment:
I image those are not important files?

Reply:
No, they are test data files. But we actually fixed this twice over:
1. They are not in stats models 0.13.1
2. wheel package v0.37.1 contains my fix for the bug https://github.com/pypa/wheel/pull/427